deployment/operator/README.md

```diff
@@ -40,12 +40,12 @@ spec:
   values:
     nri:
       plugin:
-        index: 90
+        index: 90                       # optional
       runtime:
-        patchConfig: true
-#       config:
-#         pluginRegistrationTimeout: 5s
-#         pluginRequestTimeout: 2s
+        patchConfig: true               # optional
+      config:
+        pluginRegistrationTimeout: 5s   # optional
+        pluginRequestTimeout: 2s        # optional
```

Comment:
I would change `optional` to `either neither or both of the timeouts should be given and patchConfig needs to be true for updating timeouts`.

Reply:
I removed the optional comment and added a note in about ^ below.